pyproject.toml

```diff
@@ -27,6 +27,7 @@ paho-mqtt = "~1.6.1"
 mashumaro = {version = "^3.12"}
 click = { version = "^8.1", optional = true }
 tabulate = { version = "^0.9", optional = true }
+requests = "^2.31.0"
```

Comment:
Can't we use aiohttp or httpx for this to keep the lib async?

Reply:
There are only three requests, which will be done well if once a year.
After all, there is `asyncio.to_thread`.